interfaces/builtin/avahi_control.go

```diff
@@ -123,7 +123,7 @@ func (iface *avahiControlInterface) AppArmorConnectedPlug(spec *apparmor.Specifi
 	// label like it would when installed as a snap.
 	if implicitSystemConnectedSlot(slot) {
 		// avahi from the OS is typically unconfined but known to sometimes be confined
-		new = "\"{unconfined,/usr/sbin/avahi-daemon}\""
+		new = "\"{unconfined,/usr/sbin/avahi-daemon,avahi-daemon}\""
```

Comment:
Could use a comment:
```
// with stock apparmor 2.13.2+ profiles the label is avahi-daemon
```

Reply:
Re-pushed with comment added. Thanks for tracking this down!